.env.example

```diff
@@ -0,0 +1,5 @@
+PORT=8080
+AUTH0_DOMAIN=<auth0_domain>
```

Comment:
nit: it'd be good to explain where these values can be obtained. 
I've already forgotten this...
... it transpires you can get the values from the auth0 application settings page.

Reply:
Created issue #1